Dockerfile-app

```diff
@@ -0,0 +1,28 @@
+FROM python:3.8.5-slim-buster
+WORKDIR /app
+COPY requirements.txt /app/requirements.txt
+COPY . /app
+RUN apt-get update && \
+    apt-get install -y \
+        build-essential \
+        make \
+        gcc \
+        libpq-dev \
+        libhdf5-serial-dev \
+        netcdf-bin \
+        libnetcdf-dev \
+    && pip install --upgrade pip \
+    && pip install psycopg2 \
+    && pip install -r requirements.txt \
+    && apt-get remove -y --purge make gcc build-essential \
+    && apt-get autoremove -y \
+    && rm -rf /var/lib/apt/lists/*
+
+# set some default environment variables
+ENV FLASK_ENV=prod
```

Comment:
how are we gonna set all our env variables in docker? we should be setting all the env variables related to DB, mapbox api key, slack key etc. We should set up docker secrets with github actions: https://docs.docker.com/ci-cd/github-actions/

Reply:
We are using aws ssm, see the google doc I linked in the README